README.md

```diff
@@ -142,9 +142,9 @@ mesh profile dump --set profile=minimal
 ```
 
 
-#### Select a specific configuration profile
+#### Select a specific base profile
```

Comment:
Let's stick with istio.io terminology, which is configuration profile. I also thought about using "base" but I believe it may be confusing to users.

Reply:
I think what we have right now is not clear enough though, actually the configuration profile should refer to the base+different layers of overlays
maybe change it to sth like base configuration profile?

Reply:
"configuration profile" in istio.io terminology means the built-in profiles, without any customization. We've started using "base profile" as a term in operator but for Istio users in general they understand the concept of "configuration profile" but "base" will confuse them.